libs/platform/src/lib/components/object-attribute/object-attribute.component.scss

```diff
@@ -0,0 +1,47 @@
+.fd-object-list__object-attribute {
```

Comment:
Don't we have fundamental-styles for it?
If we have or going to have it in the future please leave a corresponsing comment.

Reply:
It is getting created only for platform specific usage.